### PR TITLE
Create security page

### PR DESCRIPTION
**Pull request by drea-rodriguez**
A page that lists all Mina ecosystem-related audits, and learning material. This page was requested for ecosystem partners to easily access and find. 

We are also looking to add this page to the Mina Protocol Github (https://github.com/MinaProtocol/mina).

### CODE REVIEW

File: docs/about-mina/security [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Mina Security
+description: Check out some resources to see what measures are in place to keep Mina secure. 
+keywords:
+  - audit
+  - Ouroboros Samasika
+---
```
Comment (barriebyron) [inline docs/about-mina/security:7]:
```suggestion
description: Security audits, tools, and resources to keep Mina secure. 
keywords:
  - audit
  - Ouroboros Samasika
  - security
  - mina
---
```

File: docs/about-mina/security [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+# Mina Security
+
+Mina Protocol is built with ZK from the ground up so anyone can quickly sync and verify the network, exponentially increasing participation, true decentralization, censorship resistance, and network security.
+
+However it doesn't stop there. Check out some other resources to see what measures are in place to keep Mina secure.
```
Comment (barriebyron) [inline docs/about-mina/security:13]:
```suggestion
Mina Protocol is built with zk from the ground up so anyone can quickly sync and verify the network, exponentially increasing participation, true decentralization, censorship resistance, and network security.

However, it doesn't stop there. Check out these resources to see what measures are in place to keep Mina secure. 
```

File: docs/about-mina/security [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# Mina Security
+
+Mina Protocol is built with ZK from the ground up so anyone can quickly sync and verify the network, exponentially increasing participation, true decentralization, censorship resistance, and network security.
+
+However it doesn't stop there. Check out some other resources to see what measures are in place to keep Mina secure. 
+
+## Audits:
+**Protocol**
+- [December 13, 2023 Pickles](https://minaprotocol.com/wp-content/uploads/Least-Authority-Pickles-Final-Audit-Report.pdf)| Least Authority
```
Comment (barriebyron) [inline docs/about-mina/security:17]:
```suggestion
- [December 13, 2023 Pickles](https://minaprotocol.com/wp-content/uploads/Least-Authority-Pickles-Final-Audit-Report.pdf) | Least Authority
```

**Review by barriebyron — APPROVED** [review]
looks good @drea-rodriguez just a few tiny edits 

You can click to accept each suggestion 

After we merge this, I can create the sidebar navigation piece so the topic shows up in the docs 

HNY to you, too!

**Comment by drea-rodriguez** [comment]
Thank you, @barriebyron! 

Shall we also update this FAQ page as well where it asks "What security audits have been completed?"

**Comment by barriebyron** [comment]
> Thank you, @barriebyron!
> 
> Shall we also update this FAQ page as well where it asks "What security audits have been completed?"

Yes, imo, we want to avoid duplicate content. Consider deleting the specific content on https://docs.minaprotocol.com/about-mina/faq#what-security-audits-have-been-completed and add a link to this new document that you maintain.

**Pull request merged**